workflow-dev/tpl/deis-registry-rc.yaml

```diff
@@ -1,4 +1,3 @@
-#helm:generate helm template -o $HELM_GENERATE_DIR/manifests/deis-registry-rc.yaml -d $HELM_GENERATE_DIR/tpl/generate_params.toml $HELM_GENERATE_DIR/tpl/deis-registry-rc.yaml
```

Comment:
this is still a template just that it should be generated only when the registry_location is on-cluster.The generate logic is present in the storage.sh script.